### DISCUSSION TITLE
Sd card issue

### DISCUSSION THREAD

**Discussion by 7426hh** [bangcorrupt/hacktribe#149] [General]
Hi, i've installed Hacktribe and now find that my Electribe wont format an Sd card - i've tried around 6 cards and get the same error message - also cant save to it either. I've tried formatting on my pc but still get same message on Electribe - does anyone kniw how to fix this? Thanks

**Comment by bangcorrupt** [comment]
#147 If you create the `KORG/hacktribe` directory manually, file access should work.  

It looks like it's formatting the card correctly but only creating an `electribe sampler` directory, then checking if the `hacktribe` directory exists, causing an error.

**Comment by bangcorrupt** [comment]
Fixed in latest commit to dev branch [57685c4](https://github.com/bangcorrupt/hacktribe/commit/57685c40ab15301f7098ba5117ad473b67004f9d), thank you for testing.
